### ISSUE TITLE
Add debug log level to Mainflux logger

### ISSUE THREAD

**Issue by ghost** [absmach/magistrala#379]
Currently, the Mainflux logger is missing the debug log level. There are many cases were this level can be used. Also, add `SetLevel` method to Mainflux logger. Every service should have this logging level parameter set as environment variable `MF_<service_name>_LOG_LEVEL`.

**Comment by mfinley3** [comment]
Along with adding the debug level, I was wondering if it would be beneficial to also be able to select the log level? That way someone can select if they want mainflux to write out all of the infos or not. 

Specifically something that would look like this: 
```
| Variable             | Description                              | Default      |
|----------------------|------------------------------------------|--------------|
| MF_USERS_LOG_LEVEL   | Log level for Users (info, warn, error)  | info         | 
```
For my team and I, everything that goes into stdout gets pulled into the ELK stack for us. So it would be beneficial for us to reduce the number of success messages printed out in each component.

Let me know your thoughts on this. If it is something that sounds useful I can get a PR open asap, work has been done just have some hoops to jump through.

**Comment by anovakovic01** [comment]
This was the plan, just not written in issue description. I will update description.

**Comment by drasko** [comment]
@mfinley3 looks like we are aligned, will you be sending PR?

**Comment by mfinley3** [comment]
Yep! I will get a PR opened here ASAP. I am working on our teams path to contributing back now!

I made a few small cleanup changes to the internal PR. Going to have someone look those over and should have an open PR here tomorrow morning!

**Comment by anovakovic01** [comment]
Resolved by #402.
